lambdas/src/core/resource/cities/endpoint.rs

```diff
@@ -111,45 +240,112 @@ struct SubmissionParameters {
     pub status: Option<String>,
 }
 
-#[debug_handler]
+#[utoipa::path(
+  get,
+  path = "/cities/submissions/{submission_id}",
+  description = "Get the details of a specific sumission.",
+  tag = TAG,
+  params(
+    ("submission_id" = i32, Path, description = "Submission identifier", example = "1"),
+    ("status" = Option<str>, Query, description = "Filter for the submission status", example = "Pending"),
+  ),
+  responses(
+    (status = OK, description = "Fetches a submission", body = Submissions),
```

Comment:
should this be `Submission` or keep it pluarl? the endpoint describes a single entity.

Reply:
It is singular when it returns a single result, and plural when it returns multiple ones.

Reply:
Oh, I see! it does not match the description!

Reply:
OK, it was reversed. Fixed now. Good catch!